src/components/ComposerAttachments.vue

```diff
@@ -80,6 +77,14 @@ export default {
 			return ((uploaded / total) * 100).toFixed(1)
 		},
 	},
+	created() {
+		this.bus.$on('onAddLocalAttachment', args => {
+			this.onAddLocalAttachment()
+		})
+		this.bus.$on('onAddCloudAttachment', args => {
+			this.onAddCloudAttachment()
```

Comment:
thanks @ChristophWurst ! Implemented your changes...
only thing I wasn't sure is if I could omit those `args` somehow which aren't used

Reply:
Yes! ``this.bus.$on('onAddCloudAttachment', this.onAddCloudAttachment)``

Reply:
ahh, makes sense... thank you!